region/crateria/west.json

```diff
@@ -1283,9 +1411,31 @@
                     {"canComeInCharged": {
                       "fromNode": 1,
                       "framesRemaining": 0,
-                      "shinesparkFrames": 100
+                      "shinesparkFrames": 95,
+                      "excessShinesparkFrames": 20
```

Comment:
this breaks A

Reply:
This could get complicated with excess shinespark frames and breaking 1, 2 or all 3 sets of obstacles to save 1-2 PBs and/or to open the door before starting another spark

Reply:
Currently, A is an obstacle to using the runway which are the bottom 4 blocks in the wall.  And shinesparking does not break the bottom 2.
Intentionally ending your spark on a node would have something like `healthAtMost` and would be a separate strat if a new junction were added.  The amount of blocks power bombs break in this room is very organized: 3 left groups, 1 center group, 2 right groups, and go under the far right group.